packages/remix-react/components.tsx

```diff
@@ -19,7 +25,7 @@ import {
   useHref,
   useResolvedPath,
 } from "react-router-dom";
-import type { LinkProps, NavLinkProps } from "react-router-dom";
+import type { Merge } from "type-fest";
```

Comment:
Check warning on line 28 in packages/remix-react/components.tsx

'Merge' is defined but never used